hardware/Dummy.cpp

```diff
@@ -394,6 +396,9 @@ namespace http {
 			{
 				root["status"] = "OK";
 				root["title"] = "CreateVirtualSensor";
+				std::stringstream ss;
```

Comment:
Looks like the ID is already available in char ID[40] ?
Ni need for the stringstream and include